atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/ConflictHandler.java

```diff
@@ -20,13 +20,13 @@
      * If two transactions write to the same cell the one that was committed later will win.  We
      * are ignoring write conflicts in this case.
      */
-    IGNORE_ALL,
+    IGNORE_ALL(false, false, false),
```

Comment:
think for online upgrades you specifically need a 'lock on cells', 'lock on rows' thing for migration